insight-generator/elastic_index.sh

```diff
@@ -0,0 +1,283 @@
+                  }
+                }
+              }
+            }
+          },
+          "organizations": {
+            "type": "nested",
+            "properties": {
+              "count": {
+                "type": "long"
+              },
+              "name": {
+                "type": "string",
+                "fields": {
+                  "raw": {
+                    "type": "string",
+                    "index": "not_analyzed",
+                    "null_value": "NULL"
+                  }
+                }
+              }
+            }
+          },
+          "money": {
+            "type": "nested",
+            "properties": {
+              "count": {
+                "type": "long"
+              },
+              "name": {
+                "type": "string",
+                "fields": {
+                  "raw": {
+                    "type": "string",
+                    "index": "not_analyzed",
+                    "null_value": "NULL"
+                  }
+                }
+              }
+            }
+          },
+          "percentages": {
+            "type": "nested",
+            "properties": {
+              "count": {
+                "type": "long"
+              },
+              "name": {
+                "type": "string",
+                "fields": {
+                  "raw": {
+                    "type": "string",
+                    "index": "not_analyzed",
+                    "null_value": "NULL"
+                  }
+                }
+              }
+            }
+          },
+          "time": {
+            "type": "nested",
+            "properties": {
+              "count": {
+                "type": "long"
+              },
+              "name": {
+                "type": "string",
+                "fields": {
+                  "raw": {
+                    "type": "string",
+                    "index": "not_analyzed",
+                    "null_value": "NULL"
+                  }
+                }
+              }
+            }
+          },
+          "measurements": {
+            "type": "nested",
+            "properties": {
+              "value": {
+                "type": "long"
+              },
+              "unit": {
+                "type": "string",
+                "fields": {
+                  "raw": {
+                    "type": "string",
+                    "index": "not_analyzed",
+                    "null_value": "NULL"
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+}' "http://localhost:9200/insight-generator/docs/_mapping"
+
+# get the indices
+curl 'localhost:9200/_cat/indices?v'
+
+DIRECTORY="out"
+
+if [ -d $DIRECTORY ]; then
+  # Control will enter here if $DIRECTORY doesn't exist.
+  rm -rf $DIRECTORY
+fi
+mkdir $DIRECTORY
+python extract.py sparkler_data.json $DIRECTORY/output error
+
+cd $DIRECTORY
+
+# uploading Insight-generator output onto ElasticSearch index 
+for x in *; do
```

Comment:
this will die out if there are more than 32k things. We should use `find` and `xargs` instead here.

Reply:
Sure. I will change it. Thank you :)